AsyncDisplayKit/ASTableView.mm

```diff
@@ -1422,9 +1429,10 @@ - (void)didBeginUpdatesInRangeController:(ASRangeController *)rangeController
   _performingBatchUpdates = YES;
   [super beginUpdates];
 
-  if (_automaticallyAdjustsContentOffset) {
-    [self beginAdjustingContentOffset];
-  }
+  //TODO Reimplement _automaticallyAdjustsContentOffset
+//  if (_automaticallyAdjustsContentOffset) {
+//    [self beginAdjustingContentOffset];
+//  }
```

Comment:
This flag has a very tumultuous history, but we should really strive to support it in every version unless we make it unavailable. If it's absolutely unbearable to implement – even in a rough way – we should be sure to put in the release notes that this flag is not functional in this version.